app/_locales/en_GB/messages.json

```diff
@@ -4180,7 +4180,7 @@
   "receive": {
     "message": "Receive"
   },
```

Comment:
On develop, this is "recipientAddressPlaceholder": https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en_GB/messages.json#L4143-L4145

Reply:
I agree with you that the develop shows as posted by you.

But in v12.4.2 has this changes - https://github.com/MetaMask/metamask-extension/pull/27879/files#diff-7509d7529ea1cbc57ef79713676f7d124d9d5a446053f94836876c4cd3f3ccd6

Hence I took the v12.4.2. Now my current question is how did this code get to hotfix only?